libraries/ms-drivers/test/test_pca9539r_gpio_expander.c

```diff
@@ -308,3 +310,17 @@ void test_pca9539r_gpio_get_state_invalid_address(void) {
   Pca9539rGpioState state;
   TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pca9539r_gpio_get_state(&address, &state));
 }
+void prv_test(const struct GpioAddress *address, void *context) {
+  GpioState state = GPIO_STATE_LOW;
+  gpio_get_state(address, &state);
+}
+void test_pca9539r_gpio_subscribe_interrupts(void) {
+  GpioAddress address = { .pin = VALID_PORT_0_PIN, .port = 0 };
+  TEST_ASSERT_OK(pca9539r_gpio_subscribe_interrupts(&address, &prv_test, NULL));
+}
+
+void test_pca9539r_gpio_subscribe_interrupts_invalid(void) {
+  GpioAddress invalid_address = { .pin = INVALID_GPIO_PIN, .port = 1 };
```

Comment:
Like above, `INVALID_GPIO_PIN` is defined for an invalid _pca9539r_ pin (I admit it's kind of confusing); instead use `GPIO_PINS_PER_PORT` for an invalid pin, or use `NUM_GPIO_PORTS` for an invalid port.